src/characters/pre-universal-bob.ts

```diff
@@ -8,23 +8,23 @@ import {
 } from '@nucypher/nucypher-core';
 import { ethers } from 'ethers';
 
-import { ConditionSet } from '../conditions/condition-set';
+import { ConditionSet } from '../conditions';
 import { Keyring } from '../keyring';
 import { PolicyMessageKit } from '../kits/message';
 import { RetrievalResult } from '../kits/retrieval';
 import { base64ToU8Receiver, u8ToBase64Replacer, zip } from '../utils';
 
 import { Porter } from './porter';
 
-type decrypterJSON = {
+type PreTDecDecrypterJSON = {
   porterUri: string;
   policyEncryptingKeyBytes: Uint8Array;
   encryptedTreasureMapBytes: Uint8Array;
   publisherVerifyingKeyBytes: Uint8Array;
   bobSecretKeyBytes: Uint8Array;
 };
 
-export class tDecDecrypter {
+export class PreTDecDecrypter {
```

Comment:
Do we really want to keep this?

Reply:
I don't think we have a plan for PRE phase-out yet

Reply:
My vote would be to rip the band aid off and just get rid asap. It will annoy some adopters for sure, but we can support them switching over.

Reply:
Current development goals do not include phasing out PRE. We'll continue to support it as part of the Threshold Access Control product for the foreseeable future.

Reply:
We're not saying phase out PRE, but PRE based tDec

Reply:
Ah, got it! That was unclear from the above comments.  Well in that case let's make a plan. 😁